src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -111,12 +113,15 @@ def __init__(
                 "high_quality_fast_inference_only_refit",
                 "optimize_for_deployment",
             ],
+            "auto_stack": True,
         }
         self.kwargs = {**default_kwargs, **kwargs}
 
     def train(
         self,
         training_data: Dataset,
+        validation_data: Optional[Dataset] = None,
+        last_k_for_val: Optional[int] = None,
         eval_metric="mean_absolute_error",
```

Comment:
I think these two settings are better suited as constructor arguments, so that the signature of `train` is minimal: `train` it only gets data, and everything else is a hyperparameter and is given to the constructor. What do you think? Could you also update the class docstring to include these when you move them?